### PR TITLE
Support import of Ubuntu 20.04

### PR DESCRIPTION
**Pull request by EricEdens**
Includes:
 - Cleanup of Ubuntu's translate logic. 
 - New value for `--os` parameter available in all tools (`ubuntu-2004`)
 - Updated hint files for OVF OS detection

## Testing:
1. Ran manual import of Ubuntu 20.04 and verified SSH via the GCP web UI.
2. Ran the e2e tests that were added in the PR.

### CODE REVIEW

File: cli_tools/gce_ovf_import/ovf_utils/ovf_utils.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ var ovfOSIDToImporterOSID = map[int16]osInfo{
 	91:  {description: "TurboLinux", importerOSIDs: []string{}},
 	92:  {description: "TurboLinux 64-Bit", importerOSIDs: []string{}},
 	93:  {description: "Ubuntu", importerOSIDs: []string{}},
-	94:  {description: "Ubuntu 64-Bit", importerOSIDs: []string{"ubuntu-1404", "ubuntu-1604", "ubuntu-1804"}},
+	94:  {description: "Ubuntu 64-Bit", importerOSIDs: []string{"ubuntu-1404", "ubuntu-1604", "ubuntu-1804", "ubuntu-1804", "ubuntu-2004"}},
```
Comment (zoran15) [inline cli_tools/gce_ovf_import/ovf_utils/ovf_utils.go:160]: There are two `ubuntu-1804` now

Comment (EricEdens) [reply cli_tools/gce_ovf_import/ovf_utils/ovf_utils.go]: thanks for finding that! fixed.

**Pull request merged**